.github/ISSUE_TEMPLATE/issue_form.yml

```diff
@@ -0,0 +1,69 @@
+name: "🐞 Report a Bug"
+description: Create an issue if something does not work as expected.
+labels: ["Priority/Normal", "Type/Bug"]
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Describe the issue.
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to Reproduce
+      description: List the steps you followed when you encountered the issue. If none, type N/A.
+    validations:
+      required: true
+  - type: dropdown
+    id: component
+    attributes:
+      label: Affected Component
+      description: Select affected component.
+      options:
+        - Analytics
+        - APICTL
+        - APIM
+        - IntegrationStudio
+        - MI
+        - MIDashboard
+        - SI
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: Enter component version.
+    validations:
+      required: true
+  - type: textarea
+    id: environment
+    attributes:
+      label: Environment Details (with versions)
+      description: Mention the environment details (OS, Client, etc.) that the product is running on.
+    validations:
+      required: false
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant Log Output
+      description: Copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
```

Comment:
Do we need to add this text "This will be automatically formatted into code, so no need for backticks."